src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/attribute_editor.cljs

```diff
@@ -105,9 +105,30 @@
              :else {:success typed})))
    :render
    (fn [{:keys [props state after-update]}]
-     (let [{:keys [editing?]} props]
+     (let [{:keys [editing? writer?]} props]
        [:div {}
-        (style/create-section-header "Workspace Attributes")
+        (style/create-section-header
+          [:div {}
+           "Workspace Attributes"
+           [:span {:style {:fontSize "initial" :fontWeight "initial"}}
+            [:a {:style {:marginLeft "1em"}
+                 :href (str (config/api-url-root) "/cookie-authed/workspaces/"
+                            (:namespace (:workspace-id props)) "/"
+                            (:name (:workspace-id props)) "/exportAttributesTSV")
+                 :onClick  #(utils/set-access-token-cookie (utils/get-access-token))
+                 :target "_blank"}
+             (str "Download Attributes")]
+            (when writer?
+              [comps/Button {:text "Import Attributes"
+                             :style {:float "right" :marginTop -7}
+                             :onClick #(modal/push-modal
+                                        [modal/OKCancelForm
+                                         {:header "Import Attributes"
+                                          :show-cancel? false :ok-button {:text "Done" :onClick modal/pop-modal}
+                                          :content [:div {:style {:float "right" :width 720}}
+                                                    [import-data/Page (merge (select-keys props [:workspace-id])
+                                                                             {:reload (fn [] (modal/pop-modal) ((:request-refresh props)))}
+                                                                             {:import-type "workspace-attributes"})]]}])}])]])
         (style/create-paragraph
```

Comment:
Here, we've moved the import and export buttons from beneath the header to be in the same line as the header.
The export function appears as a link, right next to the "Workspace Attributes" header.
The import function appears as a button in the same line, but aligned to the right of the screen.
This is done so that the import/export looks the same as those for entities in the Data tab.